config/database.yml

```diff
@@ -1,7 +1,7 @@
 default: &default
   adapter: postgresql
   encoding: unicode
-  host: db
+  host: localhost
```

Comment:
@JorgeDDW  Esto lo hiciste de forma manual, o se genero automaticamente al instalar devise?

Reply:
I did it like this because I was having issue with the database, I was not using docker.